src/reducers-actions/index.ts

```diff
@@ -39,6 +40,7 @@ export interface GlobalState {
   trade: TradeState
   trades: TradesState
   localTokens: LocalTokensState
+  user: UserState
```

Comment:
Can we make the reducer only theme related? 
I understand u called user to save any UserSetting, but I think this will be biting us when we try to reuse the state in different apps. 

The whole redux and state needs a bit of better restructuring (we can do it in a later PR), but in this PR at least I would add a separate state only for the theme/reducer

Reply:
sounds good